firefox-ios/Client/Application/WindowManager.swift

```diff
@@ -208,12 +209,25 @@ final class WindowManagerImplementation: WindowManager, WindowTabsSyncCoordinato
         let result: ReservedWindowUUID
         // TODO: [9610] Unit tests should eventually be updated for these changes. Forthcoming.
         if !isIpad && !AppConstants.isRunningUnitTest {
-            // At this point we should always have either a single UUID on disk or
-            // no UUIDs because this is a brand new app install
+            // We should always have either a single UUID on disk or no UUIDs because this is a brand new app install
             if onDiskUUIDs.isEmpty {
                 result = ReservedWindowUUID(uuid: WindowUUID(), isNew: true)
             } else {
                 result = ReservedWindowUUID(uuid: onDiskUUIDs.first!, isNew: false)
+
+                let uuidCount = onDiskUUIDs.count
+                if uuidCount > 1 {
+                    // This is unexpected. Potentially related to incident in v128 (see: FXIOS-9533).
```

Comment:
```suggestion
                    // This is unexpected. Potentially related to incident in v128 (see: FXIOS-9516).
```
Linking the incident ticket, rather than a sub ticket.

Reply:
Thank you, will fix